pentestportal_ec2.tf

```diff
@@ -0,0 +1,85 @@
+# The docker AMI, which we use for our pentest portal instances
+data "aws_ami" "docker" {
+  provider = aws.provisionassessment
+
+  filter {
+    name = "name"
+    values = [
+      "docker-hvm-*-x86_64-ebs"
+    ]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  filter {
+    name   = "root-device-type"
+    values = ["ebs"]
+  }
+
+  owners      = [local.images_account_id]
+  most_recent = true
+}
+
+# The pentest portal EC2 instances
+resource "aws_instance" "pentestportal" {
+  count    = lookup(var.operations_instance_counts, "pentestportal", 0)
+  provider = aws.provisionassessment
+
+  ami                         = data.aws_ami.docker.id
+  associate_public_ip_address = true
+  availability_zone           = "${var.aws_region}${var.aws_availability_zone}"
+  iam_instance_profile        = aws_iam_instance_profile.pentestportal.name
+  instance_type               = "t3.small"
```

Comment:
Are you sure this is large enough?

Reply:
I asked @KyleEvers and he said he thinks it will work.  We can always increase it later if there are problems.